### DISCUSSION TITLE
v1.9.1 draft

### DISCUSSION THREAD

**Discussion by GeckoEidechse** [R2Northstar/Northstar#308] [Draft patch notes]
**Features:**
Add StringToAsset function to squirrel ([#216](https://github.com/R2Northstar/NorthstarLauncher/pull/216))
Add Postload support for rpaks ([#217](https://github.com/R2Northstar/NorthstarLauncher/pull/217))
Add dependency constants ([#156](https://github.com/R2Northstar/NorthstarLauncher/pull/156))

**Fixes:**
Fix KHook hooking order to prevent missed hooks ([#213](https://github.com/R2Northstar/NorthstarLauncher/pull/213))
Fix check to set loadout to callback version ([#405](https://github.com/R2Northstar/NorthstarMods/pull/405))
Require `sv_cheats` for `kill` and `explode` commands ([#414](https://github.com/R2Northstar/NorthstarMods/pull/414))
Call InitPersistentData on client connect to mirror vanilla behaviour and clear unwanted stuff ([#424](https://github.com/R2Northstar/NorthstarMods/pull/424))

**MasterServer:**
Added Dockerfile for reproducible environment ([#73](https://github.com/R2Northstar/NorthstarMasterServer/pull/73))

**Contributors:**
@anjannair @ASpoonPlaysGames @BobTheBob9 @Dinorush @EladNLG @KittenPopo @RoyalBlue1

**Comment by uniboi** [comment]
Shouldn't the versions in NorthstarMods be bumped with every minor update (even when they don't affect NorthstarMods) since the main menu displays the one in Northstar.Custom?

**Reply by GeckoEidechse** [reply]
It's not needed as the [version number is overwritten in CI on release anyway](https://github.com/R2Northstar/Northstar/blob/main/.github/workflows/build.yml#L90-L92) ^^
